CROSSSECTOR/CPSV-AP/OpeningHoursSpecification/schema.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
I guess this should be moved to opening hours specification at commons-schema.json